### ISSUE TITLE
Bulk search for collection names

### ISSUE THREAD

**Issue by gjost** [denshoproject/namesdb-editor#14]
See https://github.com/denshoproject/ddr-cmdln/issues/216

**Comment by GeoffFroh** [comment]
Error when attempting to run the `names searchmulti` command on `kyuzo`:

```
(names) ddr@kyuzo:/opt/namesdb-editor$ namesdb searchmulti /home/ddr/ddr-densho-329-creators.csv
Traceback (most recent call last):
  File "/opt/namesdb-editor/venv/names/bin/namesdb", line 33, in <module>
    sys.exit(load_entry_point('namesdb-editor==0.1.0', 'console_scripts', 'namesdb')())
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/namesdb_editor-0.1.0-py3.9.egg/names/cli.py", line 486, in searchmulti
    batch.search_multi(csvfile, prep_names, search, click)
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/namesdb_editor-0.1.0-py3.9.egg/names/batch.py", line 21, in search_multi
    for n,preferred_name,nr_id,score in search(prep_names(names)):
  File "/opt/namesdb-editor/venv/names/lib/python3.9/site-packages/namesdb_editor-0.1.0-py3.9.egg/names/batch.py", line 41, in fulltext_search_elastic
    searcher.prepare(
TypeError: prepare() got an unexpected keyword argument 'wildcards'
```

The test data file is at: `kyuzo:/home/ddr/ddr-densho-329-creators.csv`

**Comment by gjost** [comment]
Was unable to duplicate this.  `kyuzu` was running an old commit on the `develop` branch, but I got a different error when trying with the same file on my local machine.  Updated `kyuzo` and now getting a totally different error there.  Not exactly progress...

**Comment by gjost** [comment]
Fixed(?) in `develop` branch commit `6f173e5`.

**Comment by gjost** [comment]
Really actually fixed in `develop` branch commit `b8c1a66` and and pushed to `kyuzo`.
